ui/src/components/EntitySearch/FacetedEntitySearch.jsx

```diff
@@ -78,6 +88,36 @@ class FacetedEntitySearch extends React.Component {
     this.showNextPreview = this.showNextPreview.bind(this);
     this.showPreviousPreview = this.showPreviousPreview.bind(this);
     this.showPreview = this.showPreview.bind(this);
+    this.checkMobileWidth = this.checkMobileWidth.bind(this)
+    this.ref = React.createRef();
+  }
+
+  componentDidMount() {
+    window.addEventListener('resize', this.checkMobileWidth);
+    this.checkMobileWidth();
+  }
+
+  componentWillUnmount() {
+    window.removeEventListener('resize', this.checkMobileWidth);
+  }
+
+  checkMobileWidth() {
+    const width = this.ref.current?.clientWidth;
+    if (!width) { return; }
+
+    if (width < 620) {
```

Comment:
Nitpick, what is 620 here? Might it be better wrapped in a variable name

Reply:
Very valid - I'll make the change :)